Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM osrf/ros2:nightly
+SHELL ["/bin/bash", "-c"]
+
+RUN mkdir -p /opt/ros/dashing/perf_test_ws/src && \
+  cd /opt/ros/dashing/perf_test_ws/src && \
+  git clone https://github.com/ApexAI/performance_test.git
+
+RUN apt-get update && \
+  apt-get install -y \
+    nano \
+    tmux \
+    libboost-all-dev \
+    default-jre \
+    python-pip && \
+  python -m pip install \
+    pandas
+
+RUN echo "source /opt/ros/dashing/setup.bash" >> /etc/bash.bashrc && \
```

Comment:
Usually, what would happen is:
1/ Install ROS 2
2/ Create workspace and clone all custom packages using a VCS file
3/ Run rosdep to install all system dependencies
4/ Run colcon in the root workspace dir to compile everything.